RELEASE_NOTES.md

```diff
@@ -1,3 +1,9 @@
+## 3.0.1
```

Comment:
This PR adds new features, so the semver spec says it should be a minor version bump

Reply:
Updated!

Reply:
:+1: